### ISSUE TITLE
Python Notebooks no longer support relative paths

### ISSUE THREAD

**Issue by mpkarlo** [microsoft/azuredatastudio#8534]
Issue Type: <b>Bug</b>

Previously, python notebooks supported paths relative to the currently opened folder on Azure Data Studio. Since an early november update, the working path for Python is the user profile.

This broke several notebooks on my end, and I had to fix it by setting the path using "os.chdir" function. This also hampers portability of the repository, since it previously worked out-of-the-box.

Azure Data Studio version: azuredatastudio 1.13.1 (d08d07fe39c7fe74f9265db02cb963c38d048299, 2019-11-14T23:08:09.800Z)
OS version: Windows_NT x64 10.0.18362

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Xeon(R) CPU E7- 4860  @ 2.27GHz (4 x 2261)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: unavailable_off<br>protected_video_decode: unavailable_off<br>rasterization: unavailable_off<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: disabled_off<br>webgl: enabled_readback<br>webgl2: enabled_readback|
|Load (avg)|undefined|
|Memory (System)|16.00GB (9.77GB free)|
|Process Argv||
|Screen Reader|no|
|VM|100%|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
admin-pack|Mic|0.0.2
agent|Mic|0.44.0
dacpac|Mic|1.0.0
import|Mic|0.12.0
profiler|Mic|0.9.0
vscodeintellicode|Vis|1.2.1

</details>

**Comment by chlafreniere** [comment]
Hi @mpkarlo these are notebooks that you previously saved, correct? Is it not defaulting to the folder where the notebook is saved into now?

Untitled notebooks will default to a home directory.

**Comment by mpkarlo** [comment]
Hi, @chlafreniere!

Yes, these are notebooks I had created previously. They are defaulting to my profile home, and not their location or the currently open folder.

Also, I did create a new notebook that I didn't save while testing out a solution, and it also defaulted to my profile's home directory.

To be clear, they're defaulting to %userprofile% on Windows 10, which is under the username in C:/Users.

I also tested this on a co-worker's PC, since he was sharing the project with me temporarily and had the same issue. Both our PCs are virtual machines and clones, other than whatever we have installed ourselves.
